files/build_templates/init_cfg.json.j2

```diff
@@ -83,6 +83,20 @@
             "state" : "disabled", {% endif %}
             "rate_limit_interval" : "600"
         }{%if not loop.last %},{% endif -%}
-{% endfor %} 
+{% endfor %}
+    },
+    "PASSW_HARDENING": {
```

Comment:
If you add it in init_cfg.json YANG model is a must. Can you please add yang model and have the changes for init_cfg.json as part of that PR?

Reply:
still in progress working on integrating the CLI in the repo

Reply:
done